cmd/ingester/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM alpine:latest as certs
+RUN apk add --update --no-cache ca-certificates
+
+FROM scratch
+
+COPY --from=certs /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
```

Comment:
are certificates needed?

Reply:
at this point prob not, but I assume it's feasible that people might use certs to auth to Kafka.